src/handycon/handhelds/go_gen1.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+# This file is part of Handheld Game Console Controller System (HandyGCCS)
+# Copyright 2022-2023 Derek J. Clark <derekjohn.clark@gmail.com>
+
+import sys
+from evdev import InputDevice, InputEvent, UInput, ecodes as e, list_devices, ff
+
+from .. import constants as cons
+
+handycon = None
+
+def init_handheld(handheld_controller):
+    global handycon
+    handycon = handheld_controller
+    handycon.BUTTON_DELAY = 0.2
+    handycon.CAPTURE_CONTROLLER = True
+    handycon.CAPTURE_KEYBOARD = True
+    handycon.CAPTURE_POWER = True
+    handycon.GAMEPAD_ADDRESS = 'usb-0000:c2:00.3-3/input0'
+    handycon.GAMEPAD_NAME = 'Generic X-Box pad'
+    handycon.KEYBOARD_ADDRESS = 'usb-0000:c2:00.3-3/input3'
+    handycon.KEYBOARD_NAME = '  Legion Controller for Windows  Keyboard'
+
+# Captures keyboard events and translates them to virtual device events.
+async def process_event(seed_event, active_keys):
+    global handycon
+
+    # Button map shortcuts for easy reference.
+    button2 = handycon.button_map["button2"]  # Default QAM
+    button4 = handycon.button_map["button4"]  # Default OSK
+    button5 = handycon.button_map["button5"]  # Default MODE
+
+    ## Loop variables
+    button_on = seed_event.value
+
+    # Legion + x = keyboard
+    if active_keys == [99] and button_on == 1 and button4 not in handycon.event_queue:
```

Comment:
Minor nitpicking, but can you put the buttons in numerical order? This event trigger should follow button 2.

Reply:
done